ote/src/clj/ote/services/login.clj

```diff
@@ -45,27 +52,36 @@
           (encrypt/passlib->buddy (:password login-info)))
       (if (:email-confirmed? login-info)                    ;;If email not confirmed send proper error
         ;; User was found and password is correct, return user info
-        (with-auth-tkt
-          (http/transit-response
-            {:success? true
-             :session-data
-             (let [user (users/find-user db (:id login-info))]
-               (transport-operator/get-user-transport-operators-with-services db (:groups user) (:user user)))}
-            200)
-          (cookie/unparse "0.0.0.0" (:shared-secret auth-tkt-config)
-            {:digest-algorithm (:digest-algorithm auth-tkt-config)
-             :timestamp (java.util.Date.)
-             :user-id (:id login-info)
-             :user-data ""})
-          (:domain auth-tkt-config))
+        (let [current-zoned-timestamp (java-time.zone/with-zone-same-instant (java-time.zone/zoned-date-time) "UTC")]
+          (with-auth-tkt
+            (http/transit-response
+              {:success? true
+               :session-data
+               (let [user (users/find-user db (:id login-info))]
+                 (transport-operator/get-user-transport-operators-with-services db (:groups user) (:user user)))}
+              200)
+            (cookie/unparse "0.0.0.0" (:shared-secret auth-tkt-config)
+                            {:digest-algorithm (:digest-algorithm auth-tkt-config)
+                             :timestamp        (java.util.Date.)
+                             :user-id          (:id login-info)
+                             :user-data        {:session-start-timestamp current-zoned-timestamp}})
+            (:domain auth-tkt-config)
+            ;; Timestamp for cookie expiration.
+            (let
```

Comment:
IMO this is too much wrapping, just have the `8` directly in the `.plus` call.

Reply:
Fixed :)